src/types/field/fieldTypes.ts

```diff
@@ -52,3 +52,8 @@ export enum SPCPFieldTitle {
   CpUid = 'CorpPass Validated UID',
   CpUen = 'CorpPass Validated UEN',
 }
+
+export enum VerifiableFieldType {
+  Email = BasicField.Email,
+  Mobile = BasicField.Mobile,
+}
```

Comment:
nit: is it possible to `Pick` from the enum?

Reply:
that's alot better, it slipped my mind ._.

Reply:
oddly enough, `Pick` [doesn't actually work](https://github.com/microsoft/TypeScript/issues/33308). I think this is because enums are not actually types and are more akin to `const` declarations (they are values).

changing it to type and using `Extract` works but i want to refer to the sub-fields (cos of validation) rather than typing it and possibly making an error @__@

Reply:
can't it just be 
```ts
export type VerifiableFieldType = BasicField.Email | BasicField.Mobile
```
?

It will only still allow for only the two basic fields

Reply:
yeppers, the reason is cos the joi validator takes string and idw to type wrongly @__@